sample/kotlin/src/main/kotlin/com/datadog/android/sample/data/model/Log.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sample.data.model
+
+data class Log(val attributes: LogAttributes)
```

Comment:
By default Gson will reuse the attribute's name through reflection. If you obfuscate the app, it wont be able so in theory you'd need an `@SerializedName` annotation on your fields ;)